java/src/main/java/lesson05/README.md

```diff
@@ -0,0 +1,23 @@
+# Lesson 5 - Using Existing Open Source Instrumentation
```

Comment:
I prefer not to call this 'lesson05'. In all tutorials this is called 'extracredit, and there already exists a package for that, with a readme: https://github.com/yurishkuro/opentracing-tutorial/tree/master/java/src/main/java/extracredit

You can add everything there.